src/focus-trap-react.js

```diff
@@ -89,15 +89,16 @@ class FocusTrap extends React.Component {
     }
   }
 
-  setFocusTrapElement = (element) => {
+  setFocusTrapElement(element) {
```

Comment:
I think we need to _not_ do this. Afaik there are some funny things that happens when you call `this.` in js land. Think we need to make this stay as a member variable assigned a function.

Perhaps we need to invest in some testing suite things for this.

Reply:
I believe this is technically safe, because `setFocusTrapElement()` is only called from `composedRefCallback()`, which is also declared as an arrow function in `render()`, and `composedRefCallback()` calls `this.setFocusTrapElement(element)`. So it should all hang right with context. But, it doesn't seem related to the nature of the PR, so I'd be more in favor of leaving it as-is.

Reply:
If we do leave this as-is, then we'd have to revert the changes in this PR that remove the `@babel/plugin-proposal-class-properties` dependency from `package.json` and `babel.config.js`.

Reply:
@stefcameron I will revert that change. What do you think about do that on another PR? Or do you think this isn't necessary?

Reply:
Thanks, I appreciate that. I'd be open to it in another PR, especially since this is the only reason why we have that Babel dependency for class properties, and it's not really necessary. If you're up for it, please open another PR.